Tests/FoundationTests/NSUbiquitousKeyValueStoreExtensionsTests.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  NSUbiquitousKeyValueStoreExtensions.swift
+//  SwifterSwift
+//
+//  Created by Joseph Canale on 6/7/19.
+//  Copyright © 2017 SwifterSwift
+//
+
+import XCTest
+@testable import SwifterSwift
+
+#if canImport(Foundation)
+import Foundation
+
+final class NSUbiquitousKeyValueStoreExtensionsTests: XCTestCase {
+
+    private struct TestObject: Codable {
```

Comment:
Make this `Equatable` too so that you can test it more easily.
```suggestion
    private struct TestObject: Codable, Equatable {
```